src/harness/sas.py

```diff
@@ -76,6 +75,40 @@ def _Request(url, request, ssl_cert, ssl_key):
   logging.debug('Response:\n' + body)
   return json.loads(body)
 
+def _RequestGet(url, ssl_cert, ssl_key):
```

Comment:
I'll ask a different way: what's the difference in implementation between _RequestPost() and _RequestGet()? They look identical to me.